### PR TITLE
Refactoring: G->T, disease labels in the config, (UG) in the config

### PR DESCRIPTION
**Pull request by ramti**

### CODE REVIEW

File: src/ml_pipeline/config.yml [hunk 1]
```diff
@@ -9,8 +9,14 @@ default:
...
     'fs_type': ['none', 'Boruta90', 'Boruta80', 'utest'],
-    'feature_set_type': ['G','P','M','G+P','G+M','G+P+M']
+    'feature_set_type': ['T','P','M','T+P','T+M','T+P+M']
   }
+  disease_labels: ['ASD', 'CD', 'CDI', 'nonCDI', 'CIRR', 'CRC', 'EDD', 'HIV',
+    'MHE', 'NASH', 'OB', 'PAR', 'PSA', 'RA', 'T1D', 'T2D', 'UC',
+    'Carcinoma', 'Adenoma', 'hypertension', 'adenoma', 'IGT',
+    'ME/CFS', 'T2D', 'cirrhosis', 'IBD', 'migraine',
+    'otitis', 'pre-hypertension', 'schizofrenia', 'STH']
+  unknown_taxa_prefix: ['(UG)']  # features that will be removed in the preprocessing
```
Comment (efratmuller) [inline src/ml_pipeline/config.yml:19]: Just making sure: we only remove these if the feature is exactly this string right? Not any feature containing this string?

File: src/ml_pipeline/feature_name_resolver.R [hunk 2]
```diff
@@ -52,8 +52,8 @@ fnr_enrich_feature_importance <- function(dir_path) {
       left_join(joined_db, by = 'feature') %>%
       mutate(pretty_name = coalesce(pretty_name, feature))
     
-    # Fill in "G" for the unknowns feature_type, because there is no database for them
-    data$feature_type[is.na(data$feature_type)] <- 'G'
+    # Fill in "T" for the unknowns feature_type, because there is no database for them
+    data$feature_type[is.na(data$feature_type)] <- 'T'
```
Comment (efratmuller) [inline src/ml_pipeline/feature_name_resolver.R:56]: what is this for?

File: src/ml_pipeline/config.yml (hunk 1 above)
Comment (ramti) [reply src/ml_pipeline/config.yml:19]: Correct.

File: src/ml_pipeline/feature_name_resolver.R (hunk 2 above)
Comment (ramti) [reply src/ml_pipeline/feature_name_resolver.R:56]: I'm not sure it's needed anymore, legacy. I can remove it on Tuesday.

**Comment by efratmuller** [comment]
ok, thanks

**Review by efratmuller — APPROVED** [review]

**Pull request merged**